rust_src/src/windows.rs

```diff
@@ -799,4 +800,26 @@ pub fn set_window_point(window: LispObject, pos: LispObject) -> LispObject {
     pos
 }
 
+/// Make display in WINDOW start at position POS in WINDOW's buffer.
+/// WINDOW must be a live window and defaults to the selected one.  Return
+/// POS.  Optional third arg NOFORCE non-nil inhibits next redisplay from
+/// overriding motion of point in order to display at this exact start.
+#[lisp_fn(min = "2")]
+pub fn set_window_start(window: LispObject, pos: LispObject, noforce: LispObject) -> LispObject {
+    let mut w = window_live_or_selected(window);
+    set_marker_restricted(w.start, pos, w.contents());
+    // This is not right, but much easier than doing what is right.
```

Comment:
This is an amazing comment!

Reply:
Yeah. Even better than the one from #817 =)